### ISSUE TITLE
Trying to use the new beta plugin 2.1 beta 3b

### ISSUE THREAD

**Issue by GeoffMurray** [archi-contribs/database-plugin#38]
I was able to load preferences, and then when I did the database check, it indicated that the database needed to be upgraded. However, it failed.

Is there something I should do manually? Sorry if I missed something.

Using org.archicontribs.database_v2.1-beta3b.jar.

It's an MSSQL database.

![image](https://user-images.githubusercontent.com/28354503/40684243-baac0448-6345-11e8-99c1-04dd8b643d59.png)

**Comment by GeoffMurray** [comment]
Log (debug mode):

2018-05-29 13:51:50 INFO   478:DBPreferencePage               - Logger initialised.
2018-05-29 13:51:54 DEBUG   94:DBDatabaseConnection           - Opening connection to database Datacloud : driver=ms-sql, server=MIGDW1, port=1433, database=RD, schema=Archi, username=
2018-05-29 13:51:54 DEBUG  130:DBDatabaseConnection           - JDBC class = com.microsoft.sqlserver.jdbc.SQLServerDriver
2018-05-29 13:51:54 DEBUG  133:DBDatabaseConnection           - JDBC connection string = jdbc:sqlserver://MIGDW1:1433;databaseName=RD;integratedSecurity=true
2018-05-29 13:51:54 DEBUG  136:DBDatabaseConnection           - Connecting with Windows integrated security
2018-05-29 13:51:54 DEBUG  159:DBDatabaseConnection           - Will use schema Archi.
2018-05-29 13:51:54 INFO   760:DBGui                          - Please wait while checking the ms-sql database ...
2018-05-29 13:51:54 DEBUG  892:DBGui                          ┌ question : The database needs to be upgraded. You will not loose any data during this operation.
2018-05-29 13:51:54 DEBUG  892:DBGui                          │ 
2018-05-29 13:51:54 DEBUG  892:DBGui                          └ Do you wish to upgrade your database ?
2018-05-29 13:51:55 DEBUG  907:DBGui                          - answer : Yes
2018-05-29 13:51:55 DEBUG 1301:DBDatabaseConnection           - Setting database auto commit to false
2018-05-29 13:51:55 DEBUG  806:DBDatabaseConnection           - Altering table Archi.models, adding column checksum type varchar(50)
2018-05-29 13:51:55 DEBUG  174:DBDatabaseConnection           - Closing database connection.
2018-05-29 13:51:55 ERROR 1197:DBDatabaseEntryTableEditor     - Failed to check the database.
com.microsoft.sqlserver.jdbc.SQLServerException: Incorrect syntax near the keyword 'COLUMN'.
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:217)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1655)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.doExecuteStatement(SQLServerStatement.java:885)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement$StmtExecCmd.doExecute(SQLServerStatement.java:778)
	at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7505)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:2444)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:191)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:166)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeUpdate(SQLServerStatement.java:703)
	at org.archicontribs.database.connection.DBDatabaseConnection.request(DBDatabaseConnection.java:1172)
	at org.archicontribs.database.connection.DBDatabaseConnection.addColumn(DBDatabaseConnection.java:833)
	at org.archicontribs.database.connection.DBDatabaseConnection.upgradeDatabase(DBDatabaseConnection.java:881)
	at org.archicontribs.database.connection.DBDatabaseConnection.checkDatabase(DBDatabaseConnection.java:332)
	at org.archicontribs.database.preferences.DBDatabaseEntryTableEditor.checkCallback(DBDatabaseEntryTableEditor.java:1194)
	at org.archicontribs.database.preferences.DBDatabaseEntryTableEditor$6.widgetSelected(DBDatabaseEntryTableEditor.java:218)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:249)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:86)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4428)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1079)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4238)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3817)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:818)
	at org.eclipse.jface.window.Window.open(Window.java:794)
	at org.eclipse.ui.internal.dialogs.WorkbenchPreferenceDialog.open(WorkbenchPreferenceDialog.java:211)
	at org.eclipse.ui.internal.OpenPreferencesAction.run(OpenPreferencesAction.java:63)
	at org.eclipse.jface.action.Action.runWithEvent(Action.java:473)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:565)
	at org.eclipse.jface.action.ActionContributionItem.lambda$4(ActionContributionItem.java:397)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:86)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4428)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1079)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4238)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3817)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1150)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1039)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:153)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:680)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:594)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:148)
	at com.archimatetool.editor.Application.start(Application.java:81)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:653)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:590)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1499)

**Comment by GeoffMurray** [comment]
Sorry - missed that. Let me know if you'd also like me to test, Hervé. Happy to help, really like what you are doing.

**Comment by herve91** [comment]
Yes, I would be more than happy if you can stress test my plugins and alert me on all the issues without forgetting any improvement request that you may have :)

**Comment by herve91** [comment]
Hi,

I just released the beta 4 of version 2.1. It should solve your issue.

I rushed to deliver this release, and did not test all the functionalities on all the databases, so please do not hesitate to stress test it and drop me a line if you discover misbehaviours :)

**Comment by GeoffMurray** [comment]
I think I did the upgrade properly. Still have the problem. Trace is:

2018-05-31 09:27:40 INFO   477:DBPreferencePage               - Logger initialised.
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               ┌    log4j.rootLogger = TRACE, stdout, file   log4j.appender.stdout = org.apache.log4j.ConsoleAppender
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               │    log4j.appender.stdout.Target = System.out
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               │    log4j.appender.stdout.layout = org.apache.log4j.PatternLayout
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               │    log4j.appender.stdout.layout.ConversionPattern = %d{yyyy-MM-dd HH:mm:ss} %-5p %4L:%-30.30C{1} %m%n
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               │    log4j.appender.file = org.apache.log4j.FileAppender
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               │    log4j.appender.file.ImmediateFlush = true
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               │    log4j.appender.file.Append = false
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               │    log4j.appender.file.Encoding = UTF-8
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               │    log4j.appender.file.File = C:\Users\ez55q1l\DatabasePlugin.log
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               │    log4j.appender.file.layout = org.apache.log4j.PatternLayout
2018-05-31 09:27:40 TRACE  477:DBPreferencePage               └    log4j.appender.file.layout.ConversionPattern = %d{yyyy-MM-dd HH:mm:ss} %-5p %4L:%-30.30C{1} %m%n
2018-05-31 09:27:43 DEBUG   91:DBDatabaseConnection           - Opening connection to database Datacloud : driver=ms-sql, server=MIGDW1, port=1433, database=RD, schema=Archi, username=
2018-05-31 09:27:43 DEBUG  127:DBDatabaseConnection           - JDBC class = com.microsoft.sqlserver.jdbc.SQLServerDriver
2018-05-31 09:27:43 DEBUG  130:DBDatabaseConnection           - JDBC connection string = jdbc:sqlserver://MIGDW1:1433;databaseName=RD;integratedSecurity=true
2018-05-31 09:27:43 DEBUG  133:DBDatabaseConnection           - Connecting with Windows integrated security
2018-05-31 09:27:43 DEBUG  156:DBDatabaseConnection           - Will use schema Archi.
2018-05-31 09:27:43 INFO   773:DBGui                          - Checking the database structure...
2018-05-31 09:27:43 TRACE  295:DBDatabaseConnection           - Checking "Archi.database_version" table
2018-05-31 09:27:43 TRACE 1107:DBDatabaseConnection           -    > SELECT version FROM Archi.database_version WHERE archi_plugin = 'DatabasePlugin'
2018-05-31 09:27:43 DEBUG  905:DBGui                          ┌ question : The database needs to be upgraded. You will not loose any data during this operation.
2018-05-31 09:27:43 DEBUG  905:DBGui                          │ 
2018-05-31 09:27:43 DEBUG  905:DBGui                          └ Do you wish to upgrade your database ?
2018-05-31 09:27:44 DEBUG  920:DBGui                          - answer : Yes
2018-05-31 09:27:44 DEBUG 1290:DBDatabaseConnection           - Setting database auto commit to false
2018-05-31 09:27:44 DEBUG  806:DBDatabaseConnection           - Altering table Archi.models, adding column checksum type varchar(50)
2018-05-31 09:27:44 TRACE 1184:DBDatabaseConnection           -    > ALTER TABLE Archi.models ADD COLUMN checksum varchar(50) NOT NULL DEFAULT ''
2018-05-31 09:27:44 DEBUG  169:DBDatabaseConnection           - Closing database connection.
2018-05-31 09:27:44 ERROR 1183:DBDatabaseEntryTableEditor     - Failed to check the database.
com.microsoft.sqlserver.jdbc.SQLServerException: Incorrect syntax near the keyword 'COLUMN'.
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:217)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1655)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.doExecuteStatement(SQLServerStatement.java:885)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement$StmtExecCmd.doExecute(SQLServerStatement.java:778)
	at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7505)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:2444)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:191)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:166)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeUpdate(SQLServerStatement.java:703)
	at org.archicontribs.database.connection.DBDatabaseConnection.request(DBDatabaseConnection.java:1187)
	at org.archicontribs.database.connection.DBDatabaseConnection.addColumn(DBDatabaseConnection.java:833)
	at org.archicontribs.database.connection.DBDatabaseConnection.upgradeDatabase(DBDatabaseConnection.java:881)
	at org.archicontribs.database.connection.DBDatabaseConnection.checkDatabase(DBDatabaseConnection.java:315)
	at org.archicontribs.database.preferences.DBDatabaseEntryTableEditor.checkCallback(DBDatabaseEntryTableEditor.java:1180)
	at org.archicontribs.database.preferences.DBDatabaseEntryTableEditor$6.widgetSelected(DBDatabaseEntryTableEditor.java:212)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:249)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:86)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4428)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1079)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4238)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3817)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:818)
	at org.eclipse.jface.window.Window.open(Window.java:794)
	at org.eclipse.ui.internal.dialogs.WorkbenchPreferenceDialog.open(WorkbenchPreferenceDialog.java:211)
	at org.eclipse.ui.internal.OpenPreferencesAction.run(OpenPreferencesAction.java:63)
	at org.eclipse.jface.action.Action.runWithEvent(Action.java:473)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:565)
	at org.eclipse.jface.action.ActionContributionItem.lambda$4(ActionContributionItem.java:397)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:86)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4428)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1079)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4238)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3817)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1150)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1039)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:153)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:680)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:594)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:148)
	at com.archimatetool.editor.Application.start(Application.java:81)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:653)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:590)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1499)

**Comment by herve91** [comment]
My bad ... I did a mistake in the "ms-sql" string in my test :-(

I just released a new version that should work.

Else, it means that I should sleep a bit more ;-P

**Comment by GeoffMurray** [comment]
Ok - upgrade sort of looks like it worked. However, when I tried to export, not sure the upgrade was entirely successful. I ended up with this: 

![image](https://user-images.githubusercontent.com/28354503/40856592-42e49728-658d-11e8-99ac-4fef040eae4e.png)

**Comment by herve91** [comment]
Arghhh ...

What I suggest is that you please wait until my fellow developper teaches me how to put in place automated tests that I run all my use cases on all the databases brands (including SQL Server).

My apologies about that ...

**Comment by herve91** [comment]
Hi,

I corrected the SQL requests when an MS SQL Server database is selected. Plesae do not hesitate to drop me a line if you discover other issues.

Thanks and regards

**Comment by GeoffMurray** [comment]
Farther, but....

![image](https://user-images.githubusercontent.com/28354503/41069436-bc1a58c6-69a2-11e8-84e9-f0e514b32e36.png)

**Comment by GeoffMurray** [comment]
Just a thought, Hervé, but I do have use a schema called "Archi". Is there any chance some of your SQL clals are missing that prefix? I'm not sure, but I would have expected the error message above to include the Schema name in the message (ie, assuming it was truly missing).

**Comment by GeoffMurray** [comment]
Ok, I did a little debugging. I had created synonyms for some of the tables in a long ago version (prior to schema support), and didn't realize it. When I deleted all the synonyms (dbo.x pointing to archi.x), the Compare model function in database export failed much earlier. I restored all the synonyms one by one, and I could get farther and farther. So, it definitely looks like some of your DB calls are not prefixing using the schema reference.

An additional problem is one I put in all of the synonyms, I then ended up with this error:

![image](https://user-images.githubusercontent.com/28354503/41074216-c917cb12-69bb-11e8-849f-b40fc5cc0e5e.png)

**Comment by GeoffMurray** [comment]
In looking through the logs, the following calls are using/not using the schema prefix (not all call shown, just showing relevant table references):

Select from archi.database_version
Select from archi.models
Select from archi.models
Select from elements LEFT JOIN elements_in_model
Select from relationships LEFT JOIN relationships_in_model
Select from folders JOIN folders_in_model
Select from views JOIN views_in_model
Select from views_objects LEFT JOIN views_objects_in_view

**Comment by herve91** [comment]
Thanks very much for pointing out this schema prefix issue. It will be corrected in the next beta release ...

**Comment by herve91** [comment]
I just sent the release 5.4 which adds missing schema name in all SQL requests where it was missing. It won't solve all the issues you experienced but it should help.

Thanks for your tests :-)

**Comment by GeoffMurray** [comment]
Yes, I deleted my synonyms, and got to the end, where the field created_on is requested in the last query, but is actually not present in either table. 

[DatabasePlugin.log](https://github.com/archi-contribs/database-plugin/files/2082214/DatabasePlugin.log)

**Comment by GeoffMurray** [comment]
Ok - it looks like you have two queries in the compare step that are going to fail (the one that failed, and a later one) that both are looking for fields created_on in tables that don't have the fields.

views_objects
views_connections

I manually added that field to the tables, and then was able to do the compare step.

When I do the export, I now get another error. It looks like it gets all the way through generating the screen shots, and then fails on the next thing it exports:

![image](https://user-images.githubusercontent.com/28354503/41129150-ac6f61fa-6a65-11e8-99bf-8ff3028652a0.png)

**Comment by herve91** [comment]
Hi,

The new beta 6 release solves this issue.

Best regards
Hervé

**Comment by herve91** [comment]
Hi,

I just released the beta 7. I ran a bunch of tests on a MS SQL database using a specific schema and all my tests were successful.

Please do not hesitate to drop me a line if you discover other bugs.

Best regards
Hervé

**Comment by GeoffMurray** [comment]
Ok, loaded 7. Tried again. Still getting an error. Error message below. Database log included.

![image](https://user-images.githubusercontent.com/28354503/41790094-4dd30bf4-7606-11e8-821a-8e0ef53ec5f7.png)
[DatabasePlugin.log](https://github.com/archi-contribs/database-plugin/files/2128549/DatabasePlugin.log)

**Comment by GeoffMurray** [comment]
I tried adding created_by to views_objects, and tried again. Then got this error:

![screenhunter 229](https://user-images.githubusercontent.com/28354503/41815833-e75d521e-772a-11e8-8c91-8f532d102da9.png)
[DatabasePlugin.log](https://github.com/archi-contribs/database-plugin/files/2130452/DatabasePlugin.log)

**Comment by GeoffMurray** [comment]
Next error was missing created_by on views_connections. Added it also. 
Next error was that rank column in table views_connections does not allow nulls. INSERT fails. I modifed the Rank column.

Was able to export successfully.  Will be testing import now.

Was there some sort of updated database schema upgrade I should have applied first with beta 7?

**Comment by GeoffMurray** [comment]
Ok, so I closed my model after export, and then tried to re-import it. Got this:

![image](https://user-images.githubusercontent.com/28354503/41816191-3df725e8-7733-11e8-94dd-456d96538c7b.png)

[DatabasePlugin.log](https://github.com/archi-contribs/database-plugin/files/2130478/DatabasePlugin.log)

**Comment by herve91** [comment]
Thank you very much for the log files you provided.

The "invalid column name 'created_by'" is due to the fact that the database structure has not been upgraded to the latest version. I believe this is due to the fact that the migration process failed using a previous version of the plugin with MS SQL Server database.

Regarding the "rank" error, the rank column has been removed from the views_objects table in beta 5.3 (it has been replaced by a "views_objects_in_view" table.

As per the last error, the copied and past the resquest on my MS SQL database, and it ran fine. I'm using MS SQL Server 2016 Express Edition (13.0.4206). If this error happens again, I will install the same version as you to do some more testing (could you please remind me which version you've got ?).



In summary, the best way is either to run the beta plugin against a copy of you 2.0.7 datatase or against an empty one.

If you prefer acting on your existing database, you'll find a complete description of the datamodel structure [here](https://github.com/archi-contribs/database-plugin/tree/master/v2/database%20structure).

Best regards
Hervé

**Comment by GeoffMurray** [comment]
Hervé, I have kept a version of my model in git all along, so I think I'll create a new empty db, and reload the model back into that. What SQL script should I use to create those new empty tables? I wasn't quite sure.

**Comment by GeoffMurray** [comment]
Ah - ok - didn't realize that the jar does it all now by itself. That's awesome. I was able to export. More feedback shortly - I saw some other interesting behaviour, but am in meetings for a bit.

**Comment by herve91** [comment]
Great. So I close this issue. 

Please do not hesitate to open new ones if needed :-)
